### ISSUE TITLE
[Feature Request] Paste URLs and open all of them

### ISSUE THREAD

**Issue by GSPP** [hansifer/tab-copy#16]
Basically what the title says... Just paste any text into a text box, regex the URLs out and open all of them.

That then allows for moving tabs across accounts, storing into a text file and restoring them (low tech but very useful), and for migrating to this extension from other extensions that contain this feature already.

Surely, there exists a path along which to extend this simple feature when the time is right.

Seems like a logical and fitting piece.

**Comment by hansifer** [comment]
> Seems like a logical and fitting piece

Indeed. It's on the [feature roadmap](https://tabcopy.com/roadmap#display-and-open-clipboard-tabs).
